sufsara_tutorial.Rmd

```diff
@@ -1,28 +1,54 @@
+* Use the standard Session type (Xorg)
+* Your username is your SURF researchcloud username, which is not the same as your WUR username (!). You can find your SURF username in your [profile page](https://portal.live.surfresearchcloud.nl/profile)
+* Your password is the 2FA code
 
-<!--### "User" view
-Next, press "Change view" and set it to "user". From the Dashboard, go to Virtual Resources → Templates, select the "Ubuntu 16.04 GeoScripting" template and press "Clone". Give it a name you like: this will become your very own virtual machine. A *Template* is a set of settings for your virtual machine, including the number of cores, memory and hard disks.
+If your VM is coming out of 'sleep mode', and you have to fill in a password to unlock it, you should (again) make use of your 2FA code. Happy coding!
 
-You can run it already by clicking on it and pressing "Instantiate", then confirming the settings by pressing "Instantiate" again. You will see your virtual machine running in the Virtual Resources → Virtual Machines page. -->
+# Optional: accessing the VM from your own machine
+
+The tutorial above allows you to access the VM in-browser. There are however alternatives to access the VM directly from your own machine, for which we provide a tutorial below.
+
+## Enrol your SSH key
+
+Navigate to your [SRAM profile page](https://sram.surf.nl/profile). Generate an SSH key on your own machine. Paste this SSH key into the text box, and click the "Save" button. Please refer to online tutorials to find/create an SSH key for your machine, like [this one](https://www.howtogeek.com/762863/how-to-generate-ssh-keys-in-windows-10-and-windows-11/) for Windows.
+
+**Hint**: another easy way to find your SSH key is by running Git GUI on your own machine, though Help → Show SSH Key. 
 
 ## Log into the VM
 
-There are two ways to log into your virtual machine: through a graphical interface using the *X2Go Client* software, and through a command line using *Git Bash*. We will mainly use the graphical interface in this course. To do that, launch *X2Go Client* on Windows, and the new session *Session Preferences* window will be opened automatically. Fill in the details as follows:
+There are two ways to log into your virtual machine (aside from the in-browser option): through a graphical interface using the *X2Go Client* software, and through a command line using *Git Bash*. To use the graphical interface, install and launch *X2Go Client* on Windows. After launch, the new session *Session Preferences* window will be opened automatically. Fill in the details as follows:
```

Comment:
X2go is no longer supported. Instead, you can access the graphical view using the Remote Desktop protocol, either via the "Remote Desktop" app that is built-into Windows, or an RDP client on the other OSs. You simply need to specify the IP address of the VM to connect (and potentially dismiss some warnings about missing certificates).